Resync `css/compositing` from WPT upstream

Ahmad-S792 opened the pull request:
#### 71870995a4fd1ed74793646d15d8fb723137e082
<pre>
Resync `css/compositing` from WPT upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=296844">https://bugs.webkit.org/show_bug.cgi?id=296844</a>
<a href="https://rdar.apple.com/157364941">rdar://157364941</a>

Reviewed by Matt Woodrow.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/925541d35368cb5730a9d901e6bab83a86d730e9">https://github.com/web-platform-tests/wpt/commit/925541d35368cb5730a9d901e6bab83a86d730e9</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/crashtests/bgblend-root-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/crashtests/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/background-blend-mode-gradient-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/background-blend-mode-plus-lighter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/isolation-establishes-stacking-context-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/isolation-establishes-stacking-context.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph-background-image.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-with-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-stacking-context-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-stacking-context-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/green-square.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-animation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-overflow-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-with-transparent-pixels-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-with-transform-and-perspective-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-border-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-both-parent-and-blended-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-canvas-parent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-canvas-sibling-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-iframe-parent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-iframe-sibling-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-image-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-intermediate-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-mask-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-overflowing-child-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-paragraph-background-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-paragraph-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-scroll-blended-position-fixed-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-text-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-svg-basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-rotated-clip-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-script-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-sibling-with-3D-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-stacking-context-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-stacking-context-creates-isolation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-video-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-video-sibling-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-with-transform-and-preserve-3D-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/support/RGB_Circles.oggtheora.ogv: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-contain-hidden-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-opaque-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/mix-blend-mode-in-svg-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/mix-blend-mode-svg-rectangle-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/w3c-import.log:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/299675@main">https://commits.webkit.org/299675@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9332ab2133bc8ea7878fe44b30ce33c44ce544fe

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119830 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/39523 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/30174 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/126140 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71903 "Built successfully") | ⏳ 🛠 ios-apple 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/40218 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/48100 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/91011 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60300 "Passed tests") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122782 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32113 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/107451 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71567 "Passed tests") | | ⏳ 🛠 vision-apple 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31143 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25556 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/69786 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101585 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25747 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/129076 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/46750 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/35431 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/99617 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/47116 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/103642 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/99462 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25246 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44903 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22919 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/43345 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/46612 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/46078 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/49427 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/47764 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/c160ee4adbcbedd2b7d0d0037d135b2ef181db2d. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48865

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/9332ab2133bc8ea7878fe44b30ce33c44ce544fe. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48865

Ahmad-S792 committed:
Resync `css/compositing` from WPT upstream
https://bugs.webkit.org/show_bug.cgi?id=296844
rdar://157364941

Reviewed by Matt Woodrow.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/925541d35368cb5730a9d901e6bab83a86d730e9

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/crashtests/bgblend-root-change.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/crashtests/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/background-blend-mode-gradient-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/background-blend-mode-plus-lighter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/background-blending/reference/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/isolation-establishes-stacking-context-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/isolation-establishes-stacking-context.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph-background-image.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-parent-with-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-stacking-context-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-stacking-context-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/green-square.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-animation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-overflow-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-element-with-transparent-pixels-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-blended-with-transform-and-perspective-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-border-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-both-parent-and-blended-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-canvas-parent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-canvas-sibling-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-filter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-iframe-parent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-iframe-sibling-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-image-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-intermediate-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-mask-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-overflowing-child-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-paragraph-background-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-paragraph-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-paragraph.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-hidden-and-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-scroll-blended-position-fixed-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-element-overflow-scroll-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-3D-transform-ref.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/mix-blend-mode-both-parent-and-blended-with-3D-transform.html.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-border-radius-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-parent-with-text-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-svg-basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-plus-lighter-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-rotated-clip-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-script-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-sibling-with-3D-transform-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-stacking-context-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-stacking-context-creates-isolation-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-svg-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-video-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-video-sibling-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/mix-blend-mode-with-transform-and-preserve-3D-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/support/RGB_Circles.oggtheora.ogv: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/mix-blend-mode/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-contain-hidden-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-opaque-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-image-transparency-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/root-element-background-margin-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/mix-blend-mode-in-svg-image-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/mix-blend-mode-svg-rectangle-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/svg/reference/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/css/compositing/isolation/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/css/compositing/w3c-import.log:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: https://commits.webkit.org/299675@main

webkit-commit-queue commented:
Committed 299675@main (71870995a4fd): <https://commits.webkit.org/299675@main>

Reviewed commits have been landed. Closing PR #48865 and removing active labels.
